input-output-hk/lace
# [LW-9407] Quorum step storybook setup set

# Checklist

- [x] JIRA - LW-9407

---

## Proposed solution
- Add the design system component for radio button 
- Add the design system component for dropdown selectors 
- Add the view for the Quorum set up step

## Testing
- Run the storybook package in `packages/UI` and check Radio button and Select group. 
- Run storybook in `packages/core` and check the UI for the quorum step

## Screenshots
- Select component: 
<img width="1208" alt="Screenshot 2024-01-18 at 20 16 01" src="https://github.com/input-output-hk/lace/assets/4052063/1c57402b-d7ac-4fb9-938c-f1010b281ccf">
<img width="1215" alt="Screenshot 2024-01-18 at 20 16 10" src="https://github.com/input-output-hk/lace/assets/4052063/ddc7dba5-9b90-4e56-9030-cc1e2df3e21b">


- Radio button component:
<img width="1801" alt="Screenshot 2024-01-11 at 11 28 58" src="https://github.com/input-output-hk/lace/assets/4052063/0d41c840-02bc-4f1d-b1b6-32a3a4d8ddc4">
<img width="1144" alt="Screenshot 2024-01-11 at 09 16 12" src="https://github.com/input-output-hk/lace/assets/4052063/74090278-e667-4ed0-8e23-18368dd1205c">
<img width="1164" alt="Screenshot 2024-01-18 at 19 45 50" src="https://github.com/input-output-hk/lace/assets/4052063/d308bafa-bd19-4622-b5d9-5be2f1701866">

- Quorum screen:
<img width="1199" alt="Screenshot 2024-01-10 at 11 15 50" src="https://github.com/input-output-hk/lace/assets/4052063/f8d9fce7-0dc2-40a9-9d08-bd1a89cf7476">
<img width="1011" alt="Screenshot 2024-01-11 at 11 39 18" src="https://github.com/input-output-hk/lace/assets/4052063/9ababb04-8f96-4534-a4d7-2549787413b6">
<img width="1195" alt="Screenshot 2024-01-11 at 11 39 38" src="https://github.com/input-output-hk/lace/assets/4052063/97ca3288-112a-41bc-a2c8-af96a58e07e8">
<img width="1176" alt="Screenshot 2024-01-11 at 11 39 30" src="https://github.com/input-output-hk/lace/assets/4052063/3cad8683-1cc9-4e0b-aef7-8e1bd9daa752">
<img width="1003" alt="Screenshot 2024-01-11 at 11 39 23" src="https://github.com/input-output-hk/lace/assets/4052063/86e272d4-f392-43dd-a90b-1754385148f1">
